tools/cldr-apps/js/src/views/VettingSummary.vue

```diff
@@ -232,6 +232,19 @@ export default {
       return cldrLoad.getLocaleName(locale);
     },
 
+    /**
+     * Display a more user-friendly snapshot id
+     *
+     * @param id like “2022-05-16T08:15:25.083077935Z”
+     * @return like “2022-05-16 08:15 GMT”
+     */
+    humanizeSnapId(id) {
+      if (!id.match(/^\d+\-\d+\-\d+T\d\d:\d\d:\d\d\..+Z$/)) {
+        return id;
+      }
+      return id.replace("T", " ").replace(/:\d\d\..+Z/, " GMT");
+    },
```

Comment:
this implementation on the front end seemed slightly easier (for me) than using ICU to format the id on the back end; also it allows for more than one id per minute (though they're displayed the same); also it allows for further customization of the displayed id (e.g., timezone/locale-specific) without changing the underlying format in the database and http requests/responses

Reply:
format on the front end. Display the raw ID as a mouseover if needed.